### DISCUSSION TITLE
blog/2024-05-29-technical-seo-checklist

### DISCUSSION THREAD

**Discussion by giscus** [mazipan/blog-comments#23] [Q&A]
# blog/2024-05-29-technical-seo-checklist

Daftar periksa yang bisa kamu gunakan sebagai panduan terkait teknikal SEO

https://www.mazipan.space/blog/2024-05-29-technical-seo-checklist

**Comment by mazipan** [comment]
💬 Leave your comment, if you have another tips and tricks, or when you have a different point of view about the topic.

I could not list down all the things because of my limited knowledge, I might miss many things, if you think those are important and worth to know, please let me know and I will update the articles to add the missing things.

Use polite sentences, do not harm anyone by using more aggressive words. 🙏

It is okay to have different perspectives in certain area.
